torch/_inductor/codegen/cpp_utils.py

```diff
@@ -914,3 +917,73 @@ def _get_dtype_from_loopbodies(loop_bodies):
                     continue
                 dtypes.add(node.meta[OptimizationContext.key].dtype)
     return dtypes
+
+
+def _get_indexes_of_template_buf_read(
```

Comment:
These internal functions seem to be specific to `template_fusion_with_epilogues_supported`? Better to move them as local functions to it?

Reply:
Changed internal functions to be local functions.